L/LLVM_jll/Compat.toml

```diff
@@ -1,2 +1,8 @@
+[6]
+julia = "1.3.0-1.4.0"
+
+[8]
+julia = "1.4.0-1.5.0"
+
 [9]
-julia = "1"
+julia = "1.5.0-1"
```

Comment:
This was correct. It must be the same also for the other versions

Reply:
So there're no compatibility issues between LLVM_jll and Julia? I think LLVM_jll-v6.0.1 cannot be dlopen-ed in Julia-v1.4. This is based on [some old experiments](https://github.com/staticfloat/LLVMBuilder/issues/34), maybe not true anymore.

Reply:
I'm not sure what you're talking about, that link is from 2018.

All JLL packages can actually run in v1.3+, but we allow them to be installed in any v1 release

Reply:
If a JLL package cannot be `using` in some Julia versions, should we allow users to install it? Currently, LLVM_jll-v9.0.1 can be installed in Julia-v1.3, but it fails to be used:
```julia
julia> using LLVM_jll
[ Info: Precompiling LLVM_jll [86de99a1-58d6-5da7-8064-bd56ce2e322c]
ERROR: InitError: could not load library "/Users/gnimuc/.julia/artifacts/6520eab5d5ae46f4cccb6010d2390c2ebd130561/lib/libclang.dylib"
dlopen(/Users/gnimuc/.julia/artifacts/6520eab5d5ae46f4cccb6010d2390c2ebd130561/lib/libclang.dylib, 1): Symbol not found: __ZN4llvm17raw_pwrite_stream6anchorEv
  Referenced from: /Users/gnimuc/.julia/artifacts/6520eab5d5ae46f4cccb6010d2390c2ebd130561/lib/libclang.dylib
  Expected in: /Applications/Julia-1.3.app/Contents/Resources/julia/lib/julia/libLLVM.dylib
 in /Users/gnimuc/.julia/artifacts/6520eab5d5ae46f4cccb6010d2390c2ebd130561/lib/libclang.dylib
Stacktrace:
 [1] #dlopen#3(::Bool, ::typeof(Libdl.dlopen), ::String, ::UInt32) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.3/Libdl/src/Libdl.jl:109
 [2] dlopen at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.3/Libdl/src/Libdl.jl:109 [inlined] (repeats 2 times)
 [3] __init__() at /Users/gnimuc/.julia/packages/LLVM_jll/KqgSs/src/wrappers/x86_64-apple-darwin14-cxx11.jl:215
 [4] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:692
 [5] _require_from_serialized(::String) at ./loading.jl:743
 [6] _require(::Base.PkgId) at ./loading.jl:1034
 [7] require(::Base.PkgId) at ./loading.jl:922
 [8] require(::Module, ::Symbol) at ./loading.jl:917
during initialization of module LLVM_jll
```

Reply:
> If a JLL package cannot be using in some Julia versions, should we allow users to install it?

Yes

Reply:
Oh, never mind. I now know what you mean.